server/modules/activityLog/activityLogger.js

```diff
@@ -22,6 +22,7 @@ const type = {
   categoryLevelInsert: 'categoryLevelInsert',
   categoryLevelPropUpdate: 'categoryLevelPropUpdate',
   categoryLevelDelete: 'categoryLevelDelete',
+  categoryLevelsDelete: 'categoryLevelsDelete',
```

Comment:
I would still use ```categoryLevelDelete``` to keep track of each categoryLevel deleted

Reply:
it's useless to keep track of each categoryLevel deleted, you are deleting them all, you have to keep track of this